subsys/bluetooth/host/scan.c

```diff
@@ -1092,7 +1092,12 @@ int bt_le_per_adv_sync_subevent(struct bt_le_per_adv_sync *per_adv_sync,
 	struct bt_hci_cp_le_set_pawr_sync_subevent *cp;
 	struct net_buf *buf;
 
-	buf = bt_hci_cmd_create(BT_HCI_OP_LE_SET_PER_ADV_SYNC_SUBEVENT, sizeof(*cp));
+	if (params->num_subevents > 128) {
```

Comment:
do we need a define for that? what does the spec call this limit

Reply:
The spec doesnt give a name, but it is the max number of subevents allowed. Anyway, I already added a macro for it in hci.h, will update.